src/modules/jcms_migrate/src/Plugin/migrate/process/JCMSCollectionContent.php

```diff
@@ -20,6 +20,12 @@ class JCMSCollectionContent extends AbstractJCMSContainerFactoryPlugin {
   public function transform($value, MigrateExecutableInterface $migrate_executable, Row $row, $destination_property) {
     if (!empty($value)) {
       if (!isset($this->configuration['multiple']) || $this->configuration['multiple'] === FALSE) {
+        if (is_string($value)) {
+          if (strpos($value, '{') === FALSE) {
```

Comment:
Is this to check if it's JSON? If so perhaps a more elegant check would be worth it here?